ARKit+CoreLocation/POIViewController.swift

```diff
@@ -263,6 +263,21 @@ extension POIViewController {
         let theAlamo = buildViewNode(latitude: 29.4259671, longitude: -98.4861419, altitude: 300, text: "The Alamo")
         nodes.append(theAlamo)
 
+        let pikesPeakLayer = CATextLayer()
+        pikesPeakLayer.frame = CGRect(x: 0, y: 0, width: 200, height: 40)
+        pikesPeakLayer.cornerRadius = 4
+        pikesPeakLayer.fontSize = 14
+        pikesPeakLayer.alignmentMode = .center
+        pikesPeakLayer.foregroundColor = UIColor.black.cgColor
+        pikesPeakLayer.backgroundColor = UIColor.white.cgColor
+
+        _ = Timer.scheduledTimer(withTimeInterval: 1, repeats: true) { _ in
```

Comment:
I'd like to see a comment about why we're using another `Timer` instance instead of using one of the ARKit scene update callbacks. Also, I think the block reference to `pikesPeakLayer` needs to be weak.

Reply:
Good questions.

1) I didn't use one of the existing timers because this is a demo of the possibilities of a node w/ dynamic text, and I wanted to keep the callback code close to the originating source.  In a real example, the view's content will probably be changed as the result of a network fetch or some other asynchronous callback.  

2) I originally had the `[weak self]` reference, but SwiftLint (correctly) told me that it wasn't needed because the block doesn't reference self like the other callbacks do.  I tested this just now to verify there was no leak.